cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -139,14 +144,25 @@ func DeployAction(logger *logrus.Logger, versions kubermaticversion.Versions) cl
 				deployHelmBinaryFlag.EnvVar)
 		}
 
-		logger.WithFields(fields).Info("🛫 Initializing installer…")
+		var kubermaticStack stack.Stack
+		stackName := ctx.Args().First()
+		switch stackName {
+		case "kubermatic-seed":
+			kubermaticStack = kubermaticseed.NewStack()
+
+		case "kubermatic-master":
+			fallthrough
 
-		supportedProviders := kubermatic.SupportedStorageClassProviders()
-		chosenProvider := ctx.String(deployStorageClassFlag.Name)
-		if chosenProvider != "" && !supportedProviders.Has(chosenProvider) {
-			return fmt.Errorf("invalid storage class provider %q given (--%s)", chosenProvider, deployStorageClassFlag.Name)
+		default:
+			kubermaticStack = kubermaticmaster.NewStack()
 		}
 
+		if kubermaticStack == nil {
+			return fmt.Errorf("unknown stack %q specified", stackName)
+		}
```

Comment:
Can we ever reach enter this `if`? If we always default to the `kubermatic-master` stack in the switch above, do we need this?